packages/skia-draw/src/CanvasControlProvider/types.ts

```diff
@@ -63,7 +63,10 @@ export type CanvasControlContextValues = {
 export type CanvasControls = {
     undo: () => void;
     clear: () => void;
-    makeImageSnapshot: (rect?: SkRect) => SkiaDrawSnapshot | undefined;
+    makeImageSnapshot: (
+        rect?: SkRect,
+        encodingOptions?: ImageSnapshotEncodingOptions,
+    ) => SkiaDrawSnapshot | undefined;
```

Comment:
If the user wants to add `encodingOptions`, but not `rect`, they would have to do:

```ts
makeImageSnapshot(undefined, encodingOptions)
```

which is fine, but should we consider putting the arguments into an object instead?

It would be a breaking change, but I think it would be a better solution overall.

```ts
makeImageSnapshot({
  encodingOptions: someEncodingOptions,
  rect: someRect
})
```

Reply:
Yeah, I did it like this to avoid breaking changes, but if you prefer changing the signature, I'll go ahead and change it

Reply:
fixed in 627f9b7